gramps/gui/configure.py

```diff
@@ -1713,3 +1724,227 @@ def update_surn_height(self, obj, constant):
+            self.all_death_symbols = [x for x in enumerate(all_symbols)]
+            pos = config.get('utf8.death-symbol')
+            combo = self.add_combo(self.grid,
+                _('Select default death symbol'),
+                6, 'utf8.death-symbol',
+                self.all_death_symbols, callback=self.utf8_update_death_symbol, valueactive=True, setactive='')
+            combo.set_active(pos)
+            if config.get('utf8.selected-font') != "":
+                self.utf8_show_example()
+
+        return _('Genealogical Symbols'), self.grid
+
+    def can_we_use_genealogical_fonts(self, obj):
+        try:
+            import fontconfig
+            from gramps.gui.utils import ProgressMeter
+        except:
+            from gramps.gui.dialog import WarningDialog
+            WarningDialog(_("Cannot look for genealogical fonts"),
+                          _("I am not able to select genealogical fonts. "
+                            "Please, install the module fontconfig for python 3."),
+                          parent=self.uistate.window)
+            return False
+        fonts = fontconfig.query()
+        all_fonts = {}
+        symbols = Symbols()
+        nb_symbols = symbols.get_how_many_symbols()
+        self.in_progress = True
+        self.progress = ProgressMeter(_('Checking available genealogical fonts'),
+                                       can_cancel=True,
+                                       cancel_callback=self.stop_looking_for_font, parent=self.uistate.window)
+        self.progress.set_pass(_('Looking for all fonts with genealogical symbols.'), nb_symbols*len(fonts))
+        for idx in range(0, len(fonts)):
+            if not self.in_progress:
+                return
+            for rand in range(symbols.SYMBOL_MALE, symbols.SYMBOL_EXTINCT+1):
+                string = symbols.get_symbol_for_html(rand)
+                value = symbols.get_symbol_for_string(rand)
+                font = fonts[idx]
+                fontname = font.family[0][1]
+                try:
+                    vals = all_fonts[fontname]
+                except:
+                    all_fonts[fontname] = []
+                if font.has_char(value):
+                    if value not in all_fonts[fontname]:
+                        all_fonts[fontname].append(value)
+                self.progress.step()
+            for rand in range(symbols.DEATH_SYMBOL_SKULL, symbols.DEATH_SYMBOL_DEAD):
+                value = symbols.get_death_symbol_for_char(rand)
+                font = fonts[idx]
+                fontname = font.family[0][1]
+                try:
+                    vals = all_fonts[fontname]
+                except:
+                    all_fonts[fontname] = []
+                if font.has_char(value):
+                    if value not in all_fonts[fontname]:
+                        all_fonts[fontname].append(value)
+                self.progress.step()
+        self.progress.close()
+        nb1 = 0
```

Comment:
This variable is unused.

Reply:
I think it was for an old verification test. removed.